flow360/component/simulation/unit_system.py

```diff
@@ -574,22 +574,6 @@ class _TemperatureType(_DimensionedType):
     dim = u.dimensions.temperature
     dim_name = "temperature"
 
-    @classmethod
```

Comment:
Why is this removed?

Reply:
We shouldn't have a positive constrains on unit types as we discussed. This can be used for temperature offset for example

Reply:
so did you add .Positive for existing TemperatureType?